exampleCourse/questions/element/checkbox/question.html

```diff
@@ -183,14 +183,68 @@
         Part 7
     </div>
 
+    <div class="card-body">
```

Comment:
I like the new examples in the `element/checkbox` question. Could you also add another question, called something like `demo/multiAnswerMC` which demonstrates how to use `pl-checkbox` to grade in the way described in the first paragraph of #4424. That is, a checkbox question where we have 5 numerical answers, only one of which is correct, and we have `max-select = 3` and `COV` grading?